gematria/granite/python/gnn_model_base_test.py

```diff
@@ -457,8 +458,8 @@ def test_train_seq2num_encoder_decoder_model(
         learning_rate=0.01,
     )
     with mock.patch(
-        'tensorflow.compat.v1.keras.layers.LayerNormalization',
-        side_effect=tf.keras.layers.LayerNormalization,
+        'tf_keras.layers.LayerNormalization',
+        side_effect=keras.layers.LayerNormalization,
```

Comment:
If this makes the tests pass, then no change needed in this PR. But I'm wondering if we need this patch at all, and maybe we could remove it altogether?